core/arch/arm/plat-hikey/platform_config.h

```diff
@@ -38,10 +38,10 @@
 #endif /* ARM64 */
 
 /* PL011 UART */
-#if defined(CFG_HIKEY_UART3)
-#define CONSOLE_UART_BASE	0xF7113000
-#else
+#if defined(CFG_HIKEY_UART0)
```

Comment:
+1, the names sound good, but IMO that is not really relevant here. optee_os only knows about secure world and uses only one console, so I see no reason to use a name other than `CFG_CONSOLE_UART`. Then, higher-level projects that have a notion of a secure and non-secure console (i.e., build.git) can just do `make -C optee_os CFG_CONSOLE_UART=$(CFG_SW_CONSOLE_UART)`.